.travis.yml

```diff
@@ -6,7 +6,7 @@ php:
 
 before_script:
   - sh -c "pear channel-discover pear.cakephp.org"
-  - sh -c "pear install PHP_CodeSniffer"
+  - sh -c "pear install PHP_CodeSniffer-1.4.8"
```

Comment:
Composer should be used instead of pear to get the phpcs. You can check the [.travis.yml](https://github.com/FriendsOfCake/crud/blob/cake3/.travis.yml) of Crud plugin for reference.